lte/gateway/python/magma/mobilityd/tests/ip_alloc_dhcp_test.py

```diff
@@ -82,8 +82,7 @@ def test_ip_alloc(self):
         sid1 = "IMSI02917"
```

Comment:
same here for the name of the test file, `test_ip_alloc_dhcp.py`

Reply:
I think it should be separate change to fix directory structure and file name.

Reply:
Sounds good, let's keep it in mind and add it as TODO.